src/translations/nb_NO.json

```diff
@@ -66,7 +66,7 @@
   "CHECKOUT_EMAIL_PLACEHOLDER": "din.epost@her.no",
   "CHECKOUT_SUMMARY_TITLE": "Dine detaljer",
   "CHECKOUT_SIGN_BUTTON_TEXT": "Signer med BankID",
-  "CHECKOUT_SIGN_DISCLAIMER": "Ved å trykke på ”Signer med BankID” godkjenner jeg at jeg har tatt del av [informasjon før kjøpet]({PREBUY_LINK}) og [vilkårene]({TERMS_LINK}) og at personopplysningene mine [behandles i tråd med GDPR](https:\/\/www.hedvig.com\/privacy).",
+  "CHECKOUT_SIGN_DISCLAIMER": "",
```

Comment:
is this intentional? :(

Reply:
Gustav already commented on this and in the spirit of having "real" translations there is another text key called `CHECKOUT_SIGN_DISCLAIMER_NO`.